CMakeLists.txt

```diff
@@ -115,6 +119,16 @@ if(NOT "${QUIC_LOGGING_TYPE}" MATCHES "^(etw|lttng|stdout|)$")
     message(FATAL_ERROR "QUIC_LOGGING_TYPE must be \"etw\", \"lttng\" or \"stdout\", not \"${QUIC_LOGGING_TYPE}\"")
 endif()
 
+if (QUIC_ENABLE_SANITIZERS OR QUIC_ENABLE_ASAN OR QUIC_ENABLE_LSAN OR QUIC_ENABLE_UBSAN)
```

Comment:
I would simplify the rest of the code by setting QUIC_ENABLE_ASAN, QUIC_ENABLE_LSAN and QUIC_ENABLE_UBSAN to TRUE here if QUIC_ENABLE_SANITIZERS is set, and then not have to check the different combinations.

Although there might be "alignment" missing? To check

Reply:
Looking more, I think I'd add an extra variable for "all other sanitizers".
Then, after you set all your variables here to the right value based on the inputs, the logic can be a simple "if (X) enable Y", without `else` or `OR`.

Reply:
Sounds good to me. Done